3p/twitter.js

```diff
@@ -59,19 +59,47 @@ export function twitter(global, data) {
         // We listen for resize to learn when things are
         // really done.
         iframe.contentWindow.addEventListener('resize', function() {
-          render();
+          resize(iframe.contentWindow.document.body);
         }, true);
-        render();
+        resize(iframe.contentWindow.document.body);
+      } else if (global.MutationObserver) {
+        // If no iframe is created, Twitter likely went into their ShadowDOM
+        // based pass. MutationObserver should definitely be available then.
+        // Because Twitter does not actually provide an event for when
+        // rendering is done, we use a MutationObserver and schedule a measure
+        // operation whenever any elements were added to the DOM.
+        let timeout;
+        function measure() {
+          // Throttle if multiple things happen at once.
+          clearTimeout(timeout);
+          timeout = setTimeout(function() {
```

Comment:
In their code I saw things like injecting `<style>` elements into shadow root with many styles.